### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by ak47na**
Week1 Walkthrough

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -13,16 +13,33 @@
...
+  const imgUrl = 'images/img' + imageIndex + '.jpeg';
+
+  const imgElement = document.createElement('img');
+  imgElement.src = imgUrl;
+
+  const imageContainer = document.getElementById('random-image-container');
+  // Remove the previous image.
+  imageContainer.innerHTML = '';
+  imageContainer.appendChild(imgElement);
+}
```
Comment (LiboShen) [inline portfolio/src/main/webapp/script.js]: nit: usually people put a newline in the end of a file.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -13,8 +13,7 @@
 // limitations under the License.
 
 /**
- * Generates a URL for a random image in the images directory and adds an img
- * element with that URL to the page.
+ * Adds a random quote to the page.
```
Comment (LiboShen) [inline portfolio/src/main/webapp/script.js]: nit: "adds" makes me think every time this function is called, a new quote is added to the page. It's not the case, however. There will be at most one quote shown. So maybe just say "shows a random quote in the page" or something similar?

**Review by LiboShen — CHANGES_REQUESTED** [review]

**Comment by LiboShen** [comment]
Generally look great. I just make some minor comments.

File: portfolio/src/main/webapp/placesGallery.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    <div id="content">
+      <h1>Favourite Places Gallery</h1>
+      <div class="image-thumbnails">
+        <a href="images/place1.jpg"><img src="images/place1.jpg"/></a>
+        <a href="images/place2.jpg"><img src="images/place2.jpg"/></a>
+        <a href="images/place3.jpg"><img src="images/place3.jpg"/></a>
+      </div>
+    </div>
+  </body>
+</html>
```
Comment (LiboShen) [inline portfolio/src/main/webapp/placesGallery.html]: nit: newline EoF :)

There are tools can help you to take care of these tedious consistency check, they are called linters.
For example, you can take a look of https://prettier.io/. This is pretty popular in the JavaScript Community.

**Review by LiboShen — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/placesGallery.html (hunk 3 above)
Comment (ak47na) [reply portfolio/src/main/webapp/placesGallery.html]: Thank you, this is very helpful!

**Review by LiboShen — APPROVED** [review]
👍

**Pull request merged**